fab/operations/supervisor.py

```diff
@@ -277,10 +278,32 @@ def restart_all_except_webworkers():
     _services_restart()
 
 
+@roles(ROLES_STATIC)
+def _decommission_host(host):
+    files.comment(
+        '/etc/nginx/sites-available/{}_commcare'.format(env.environment),
+        host,
+    )
+    sudo('nginx -s reload')
+
+
+@roles(ROLES_STATIC)
+def _recommission_host(host):
+    files.uncomment(
+        '/etc/nginx/sites-available/{}_commcare'.format(env.environment),
+        host,
+    )
+    sudo('nginx -s reload')
+
+
 @roles(ROLES_DJANGO)
 @serial
 def restart_webworkers():
+    host = env.host
+    _decommission_host(host)
     _services_restart()
+    _recommission_host(host)
```

Comment:
i went back and forth on whether or not to include this in a `finally` block. on one hand you might not want to recommission a node that has failed to restart. on the other hand you might end up in a weird state if you don't uncomment it. i'm leaning towards adding a `finally` block, but open to ideas